src/core/layout/qgslayouttable.cpp

```diff
@@ -612,31 +697,62 @@ void QgsLayoutTable::setShowEmptyRows( const bool showEmpty )
 
 void QgsLayoutTable::setHeaderFont( const QFont &font )
 {
-  if ( font == mHeaderFont )
```

Comment:
@nyalldawson , any reason why you're not doing a font == mHeaderFont.toQFont() check here?

Reply:
I'd also have to check the logic regarding size units too.. seems like overkill for a deprecated method which isn't used anywhere anymore!

Reply:
@nyalldawson , fair enough.